ports/supergaelicdodgeball/gameinfo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./Super Gaelic Dodgeball.sh</path>
+    <name>Super Gaelic Dogdeball</name>
+    <desc>Mildly violent Dodgeball game made for Awful Winter Jam 2018</desc>
+    <releasedate>20220122T000000</releasedate>
+    <developer>Shoehead</developer>
+    <publisher>Icth.io</publisher>
```

Comment:
Typo, and is itch really the publisher or just the hoster?
Hoster is not publisher